build/ci.go

```diff
@@ -329,18 +329,22 @@ func doLint(cmdline []string) {
 	// Run fast linters batched together
 	configs := []string{
 		"--vendor",
+		"--tests",
 		"--disable-all",
+		"--enable=goimports",
+		"--enable=varcheck",
 		"--enable=vet",
 		"--enable=gofmt",
 		"--enable=misspell",
 		"--enable=goconst",
+		"--deadline=5m",
```

Comment:
Please don't add a deadline here. Only fast linters should be included in this batch that finish in a couple seconds. For slower linters, there's a second pass below.

Reply:
Travis build fails randomly and report false alerts. So I increased the deadline to avoid having a false positive.

Reply:
Please don't. The point of these linter steps is to run (and fail) fast. Any linter step requiring more than a few secs should be placed below, in the slow set.

Reply:
Is Travis that is slow. Is a shared env. 
But ok, as you wish.